nise/__main__.py

```diff
@@ -267,7 +267,15 @@ def add_ocp_parser_args(parser):
         required=False,
         help="URL for Insights Upload Service.",
     )
+    parser.add_argument(
+        "--ros-ocp-info",
+        metavar="INSIGHTS_ROS_OCP_INFO",
+        dest="ros_ocp_info",
+        required=False,
+        type=str,
+        help="Provide Yes/yes value to get ROS for openshift data",
 
+    )
```

Comment:
```suggestion
    parser.add_argument(
        "--ros-ocp-info",
        metavar="INSIGHTS_ROS_OCP_INFO",
        dest="ros_ocp_info",
        required=False,
        action="store_true",
        help="Generate ROS for Openshift data",
    )
```
This should be a boolean flag. Using `Yes/yes` is error prone.